### PR TITLE
IDEM-193

### PR DESCRIPTION
**Pull request by xiangzhe0812**

### CODE REVIEW

**Comment by xiangzhe0812** [comment]
Can ignore the sonar cloud security hotspots, I put basic auth password in the config file.

File: app/apis/Api.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { UserSignup } from "../src/types/user";
+import HTTPClient from "./HTTPClient";
+export default class Api extends HTTPClient {
+  public gpibSingup = async (body: UserSignup) =>
```
Comment (bitcoinbrisbane) [inline app/apis/Api.ts:4]:
```suggestion
  public gpibSignup = async (body: UserSignup) =>
```

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
Typo

**Review by amagon270 — DISMISSED** [review]

File: app/src/context/Exchange.tsx [hunk 2]
```diff
@@ -150,21 +143,22 @@ export const ExchangeProvider: React.FC<{
...
       if (checkUserAuth.status === 200) {
         const jwt = checkUserAuth.data.token;
-        const updatedBody = {
+        const userInfo = {
           firstName: body.firstName,
           lastName: body.lastName,
           yob: Number(body.yob)
         };
-        const updateUserInfo = await axios.put(
+        console.log(userInfo);
```
Comment (pondpiu) [inline app/src/context/Exchange.tsx:151]: log

**Review by pondpiu — APPROVED** [review]
LGTM 🥔

**Review by JasminFarhi — APPROVED** [review]

**Review by bitcoinbrisbane — APPROVED** [review]

**Pull request merged**